components/ws-manager-mk2/pkg/constants/constants.go

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package constants
+
+// ManagedBy identifies who is managing the workspace
+const ManagedBy = "ws-manager-mk2"
```

Comment:
What motivates making this its own package rather than a constant in the controllers package?

Reply:
We use it outside the controller package as well (in manager.go)